components/conversations/CreateConversationForm.js

```diff
@@ -166,12 +188,15 @@ const CreateConversationForm = ({ collective, LoggedInUser, suggestedTags, onSuc
                 <LoadingPlaceholder height={38} />
               ) : (
                 <StyledInputTags
+                  name="tags"
+                  {...formik.getFieldProps('tags')}
                   maxWidth={300}
                   suggestedTags={suggestedTags}
-                  value={conversationTags}
-                  onChange={options =>
-                    setValue('tags', options && options.length > 0 ? options.map(option => option.value) : [])
-                  }
+                  onChange={options => {
+                    const tags = [];
+                    options && options.length > 0 ? options.map(option => tags.push(option.value)) : null;
```

Comment:
I think the `null` here should be replaced with `[]` because we want to keep this consistent with the default value for tags.  This could also cause errors with the length check for tags on line 100 and 108.

Reply:
You’re right, thanks, I didn’t notice that.